CHANGELOG.rst

```diff
@@ -26,13 +26,14 @@ Bug Fixes
 """""""""
 
 - dataOrder: mesh attribute is a string #355
+- constant scalar MeshRecords: reading corrected #358
```

Comment:
note: also scalar ParticleRecords

Reply:
The logic for `ParticleRecords` was actually correct, it just did manually what the second `READ_PATH` now does automatically. It mainly is there to simplify and unify the code in both.